tests/06-lb.sh

```diff
@@ -384,13 +304,13 @@ cilium service delete 5 || {
 # We can also use multiple --backends that will get appended.
 SVC_IP6="f00d::1:1"
 cilium service update --rev --frontend "[$SVC_IP6]:0" --id 222 \
-                        --backends "[$SERVER1_IP]:0" \
-                        --backends "[$SERVER2_IP]:0"
+                        --backends "$SERVER1_IP" \
```

Comment:
It's testing out ability to parse without the port specified. It's an L3 load balancing entry.